### DISCUSSION TITLE
Question about Rattlesnake capabilities

### DISCUSSION THREAD

**Discussion by robfairh** [idaholab/moose#17167] [Q&A Modules: General] [answered]
Hi everyone,

I have the following question: Does rattlesnake solve the PN or the SPN equations? 

Thanks!

P.S.: In case of being capable of solving both, are both options available to the users? Because I haven't seen anything about SPN on the [theory](https://rattlesnake.inl.gov/SiteAssets/SitePages/manuals/theory.pdf)/[user](https://rattlesnake.inl.gov/SiteAssets/SitePages/manuals/manual.pdf) manuals.

**Comment by YaqiWang** [comment, marked as answer]
No SPN is supported by Rattlesnake. And we do not have plan to support it soon. Currently only PN or SN, diffusion are available to the users.

**Reply by robfairh** [reply]
Thanks!
